src/pageServices/connector.ts

```diff
@@ -179,7 +179,7 @@ export const sendMessage = {
   checkSession: (payload: null, onResponse?: () => void): Promise<{ message: string; tabId: number }[]> =>
     connector.sendMessageToAllTabs("CHECK_SESSION", payload, onResponse),
   defineTabId: (payload: number) => connector.sendMessage("DEFINE_TAB_ID", payload),
-  evaluateXpath: (payload: { xPath: string; originJdnHash?: string }, onResponse?: () => void) =>
+  evaluateXpath: (payload: { xPath: string; originJdnHash?: string, element_id?: ElementId }, onResponse?: () => void) =>
```

Comment:
I believe we also need to update params name in utils.ts in contentScripts directory